app/views/grantees/show.html.erb

```diff
@@ -10,6 +10,7 @@
       <div class="summary-section">
         <div class="ct-section-divider">
           <h2>Summary</h2>
+          <a href="<%= @grantee.hses_link %>">Open in HSES</a>
```

Comment:
[question] Do we want this to be opened in a new tab / window?  Would that make the most sense for a program specialist?